### PR TITLE
[SIM_FLUTTER-126] [REFACTOR] Resolve TODO: Get the auth token form BE

### PR DESCRIPTION
**Pull request by egiegarciano14**
### Issue Links
- [backlog](https://framgiaph.backlog.com/view/SIM_FLUTTER-126)
- [slack](https://sun-philippine.slack.com/archives/C059L7XAXFB/p1689761879188719)

###  Notes
- To QA, pls check if the header is displaying correctly

### Definition of done
- [x] removed prop hasAuthToken

### CODE REVIEW

File: frontend/lib/src/controllers/auth_header_controller.dart [hunk 1]
```diff
@@ -15,4 +18,10 @@ class AuthHeaderController extends GetxController {
 
     return name;
   }
+
+  Future<bool> getLoginToken() async {
```
Comment (jstephend-sun) [inline frontend/lib/src/controllers/auth_header_controller.dart]: rename to something else, its not getting the login token, 
previous prop `hasAuthToken`  is ok or hasLoginToken

File: frontend/lib/src/controllers/auth_header_controller.dart [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import 'package:get/get.dart';
 import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 
+import 'package:frontend/src/services/user_service.dart';
 import 'package:frontend/src/helper/storage.dart';
 
 class AuthHeaderController extends GetxController {
   final RxString _name = ''.obs;
+  final RxBool _loginToken = false.obs;
```
Comment (jstephend-sun) [inline frontend/lib/src/controllers/auth_header_controller.dart]: rename to truthy variable name

**Review by jstephend-sun — CHANGES_REQUESTED** [review]
Requested Changes #81

review time: 10mins
comments: 2

**Review by jstephend-sun — APPROVED** [review]
Approved #81

review time: 10mins
comments: 2
iteration: 1st review

**Review by Dbulawan13 — APPROVED** [review]
Approve PR #81

Working:
- The header part is still okay.
- It is able to display a dynamic name.
- If the name is too long, it will be truncated.
- Once the user updates their user profile, the name part in the header will also be updated in real-time.

https://github.com/framgia/sph-flutter/assets/86758474/3d417101-c831-4c0b-a50f-e296bfad12f6

Testing Hours: 1
Comment: 0
Iteration: 0

**Pull request merged**